rest/rest-api/src/main/java/org/ow2/proactive_grid_cloud_portal/common/SchedulerRestInterface.java

```diff
@@ -1539,6 +1539,31 @@ JobIdData submitFromUrl(@HeaderParam("sessionid") String sessionId, @HeaderParam
             throws JobCreationRestException, NotConnectedRestException, PermissionRestException,
             SubmissionClosedRestException, IOException;
 
+    /**
+     * Submits a workflow to the scheduler from a workflow URL, creating hence a
```

Comment:
u have copied/pasted the endpoint description from the previous one, we should explain the difference between the original endpoint and the new one in the description

Reply:
the other 2 endpoints have the same description. In fact, they do the same thing, just the parameters are different. I have changed the parameters description:
     * @param jsonBody
     *            a json with the variables of the workflow

Reply:
ok.thx